go/encoding/basex/base64_compat_test.go

```diff
@@ -0,0 +1,35 @@
+package basex
+
+import (
+	"encoding/base64"
+	"testing"
+)
+
+const encodeURL = "ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz0123456789-_"
+
+func newBase64URLEncoding() *Encoding {
+	return NewEncoding(encodeURL, 3)
+}
+
+func TestHelloWorld(t *testing.T) {
+	encoding := newBase64URLEncoding()
+	input := []byte("Hello world! It is I, Bubba Karp!")
+	for i := 0; i <= len(input); i++ {
+		for j := 0; j <= 20; j++ {
```

Comment:
By the way, I know a lot of my comments boil down to "use longer variable names for this." If you would like fewer comments like that, let me know :)